server/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
 
 services:
   db:
-    image: postgis/postgis:13-3.1
+    image: odidev/postgis:13-3.1
```

Comment:
Why?

Reply:
Steht im commit: weil die im original repo kein arm anbieten. frisst dann aber zu viel Akku bei mir in der emulation.